src/main.zig

```diff
@@ -66,6 +66,11 @@ pub fn main() anyerror!u8 {
     } else {
         if (stdin.isTty()) {
             try choices.readAll();
+        } else if (blk: {
+            const stat = try std.os.fstat(stdin.handle);
+            break :blk std.os.S.ISREG(stat.mode);
```

Comment:
i'm not sure if this `ISREG` is available in other system

Reply:
Looks like it's at least available on all of the systems `fzy.zig` supports.